vagrant_provision.sh

```diff
@@ -47,15 +52,19 @@ cat /home/vagrant/rdfs/config/hdfs-site.xml > /home/vagrant/hadoop3/etc/hadoop/h
 cat /home/vagrant/rdfs/config/core-site.xml > /home/vagrant/hadoop3/etc/hadoop/core-site.xml
 
 # add hadoop to path
-echo 'export PATH=/home/vagrant/hadoop2/bin:$PATH' >> /home/vagrant/.bashrc
+echo 'export PATH=${PATH//:\/home\/vagrant\/hadoop2\/bin:/:}' >> /home/vagrant/.bashrc
```

Comment:
If the desired version of hadoop is symlinked to `$HOME/hadoop`, what is the purpose of this line?

Reply:
So, if you don’t destroy and up, then the path will still have hadoop2. That line is to get rid of it for people who are just re-provisioning.

Reply:
Cool makes sense